rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -350,6 +350,7 @@ __rmw_create_node(
             security_logging_file_path,
             property_policy.properties()))
         {
+          RMW_SET_ERROR_MSG("couldn't parse security logging config file!");
```

Comment:
This is left off on purpose. RMW can only have one error message. By setting it here you overwrite the more informative messages already set that caused this function to return `false` in the first place.

Reply:
Oh ok my bad. Gonna remove the commit.